spec/factories/application_qualification.rb

```diff
@@ -6,18 +6,26 @@
     subject { Faker::Educator.subject }
     grade { %w[A B].sample }
     predicted_grade { %w[true false].sample }
-    start_year { Time.zone.today.year }
-    award_year { Faker::Date.between(from: 60.years.ago, to: 3.years.from_now).year }
+    award_year { qualifications_start_year + 3 }
     institution_name { Faker::University.name }
     institution_country { Faker::Address.country_code }
     equivalency_details { Faker::Lorem.paragraph_by_chars(number: 200) }
 
+    transient do
+      qualifications_duration { 6 }
+      year_seed { Time.zone.now.sec - qualifications_duration }
+      qualifications_start_year { Time.zone.now.year - year_seed }
+      degree_start_year { qualifications_start_year + qualifications_duration }
+    end
+
     factory :gcse_qualification do
       level { 'gcse' }
       qualification_type { 'gcse' }
       subject { %w[maths english science].sample }
       grade { %w[A B C].sample }
       predicted_grade { false }
+      start_year { qualifications_start_year }
```

Comment:
do GCSEs have a start year?

Reply:
no – as a result of this I've also had to update the [ApplicationDataExport](https://github.com/DFE-Digital/apply-for-teacher-training/pull/6118/commits/b3cccf85fddc0ef808d19b019054c4c1cd84eeea)